indexer-balances/Dockerfile

```diff
@@ -3,10 +3,14 @@ WORKDIR /tmp/
 
 # this build step will cache your dependencies
 COPY Cargo.toml Cargo.lock ./
-RUN mkdir src && touch src/lib.rs && cargo build --release && rm -r src
+COPY indexer-base indexer-base
+COPY indexer-accounts indexer-accounts
+COPY indexer-events indexer-events 
```

Comment:
```diff
- COPY Cargo.toml Cargo.lock ./
- COPY indexer-base indexer-base
- COPY indexer-accounts indexer-accounts
- COPY indexer-events indexer-events 
+ COPY Cargo.lock ./
+ RUN echo '[workspace]\nmembers = ["indexer-balances"]' > Cargo.toml
```
Rather than copying every workspace member to the docker image, we can construct a new `Cargo.toml` with the single member we want to build. This is better for two reasons:
1. As the repository/workspace grows, we don't have to update every `Dockerfile`
2. Changes made to other members/indexers don't affect this `Dockerfile`s cache

What do you think?

Reply:
We'd also need to add any library crates the binary depends on, so the `echo ... > Cargo.toml` could grow, but I think this is still better than copying in everything 🤔

Reply:
Nice, updated.